Makefile.toml

```diff
@@ -720,23 +720,20 @@ if ! ${TMUX} ls &>/dev/null ; then
   exit 0
 fi
 
-# Kill other components
+# Kill other components with Ctrl+C/Ctrl+D
 ${TMUX} list-windows -F "#{window_name} #{pane_id}" \
-| grep -v 'kafka' \
-| grep -v 'zookeeper' \
+| grep --invert-match --extended-regexp '(kafka|zookeeper)' \
 | awk '{ print $2 }' \
 | xargs -I {} ${TMUX} send-keys -t {} C-c C-d
 
+# Kill kafka cleanly. Ctrl+C will lose data.
 if [[ -n $(${TMUX} list-windows | grep kafka) ]];
 then
   echo "kill kafka"
   kill_kafka || true
 
   echo "kill zookeeper"
   kill_zookeeper || true
-
-  # Kill their ${TMUX} sessions
-  ${TMUX} list-windows -t risedev -F "#{pane_id}" | xargs -I {} ${TMUX} send-keys -t {} C-c C-d
 fi
 
 ${TMUX} kill-server
```

Comment:
We can do a post-check here to see if kafka / zookeeper / rw nodes are still running. If yes, we can exit 1.

Reply:
I guess we can use `C-c` (to all windows) to replace `kill-server` to achieve this.

Reply:
BTW we have `set -e`, so the `test` after `kill-server` doesn't work..